lib/foreman_ansible/engine.rb

```diff
@@ -30,9 +31,20 @@ class Engine < ::Rails::Engine
           permission :play_roles,
                      { :hosts => [:play_roles, :multiple_play_roles] },
                      :resource_type => 'Host::Managed'
+          permission :view_ansible_roles, :ansible_roles => [:index],
+                                          :hosts => [:new, :edit]
+          permission :destroy_nasible_roles, :ansible_roles => [:destroy]
+          permission :edit_ansible_roles,
+                     :ansible_roles => [:import, :confirm_import]
```

Comment:
Mind to call this permission `:import_ansible_roles`? It's just for consistency so it matches with `import_puppetclasses` and `import_environments` permission